tests/test_polydata.py

```diff
@@ -857,3 +857,19 @@ def test_n_verts():
 def test_n_lines():
     mesh = pyvista.Line()
     assert mesh.n_lines == 1
+
+
+def test_geodesic_error_message(sphere, sphere_shifted):
+
+    combined = (
+        sphere + sphere_shifted
+    )  # the sphere and sphere_shifted are disconnected - no path between them
```

Comment:
It would be nice to get this on one line. Either move the comment before the assignment, or just rephrase to something like
```suggestion
    combined = sphere + sphere_shifted  # disconnected meshes
```

Reply:
I've moved the comment above the assignment